insights/parsers/nmcli.py

```diff
@@ -178,7 +178,10 @@ class NmcliConnShow(CommandParser):
 
     """
     def parse_content(self, content):
-        self.data = parse_delimited_table(content)
+        try:
+            self.data = parse_delimited_table(content, heading_ignore=["NAME", "UUID", "TYPE", "DEVICE"])
+        except:
+            raise SkipException("Inavalid Contents!")
```

Comment:
@vishwanathjadhav a minor spelling error, I believe this should be "Invalid".

Reply:
Done